raphtory/src/db/graph/views/filter/model/property_filter.rs

```diff
@@ -1105,6 +1283,22 @@ impl<M: Send + Sync + Clone + 'static> InternalPropertyFilterOps
     }
 }
 
+impl<M> AnyTemporalPropertyFilterBuilder<M> {
+    pub fn any(self) -> AnyElemFilterBuilder<M> {
```

Comment:
I think you can make this a trait and blanket-implement it for anything that is InternalPropertyFilterOps?